src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,17 +65,18 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
-		$this->params = shortcode_atts( array(
-			'limit' => -1,
-			'user'  => get_current_user_id()
-		), $params, $this->shortcode_name );
+		$this->params = shortcode_atts( [
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 6371ec85c545f414d4f56eeedb87eb35</sub>